preemptive_requests.py

```diff
@@ -133,6 +136,48 @@ def setup_requests(self):
                 )
                 self.backcompat.add_ev_ranges_alternate(row_ev_bytes)
 
+                # Special handling for showing scaled transmission requests
+                # from maximum credible energy judgement factor
+                trans_label = widget.findChild(
+                    PyDMLabel,
+                    "transmission_label"
+                )
+                jf_trans_label = widget.findChild(
+                    QtWidgets.QLabel,
+                    "jf_trans_label",
+                )
+                # Make it easy to look up past values in callbacks without
+                # holding extra references and mucking up gc
+                # Also makes the table widget items work
+                jf_trans_label.channel = trans_label.channel
+                jf_trans_channel = PyDMChannel(
+                    trans_label.channel,
+                    value_slot=functools.partial(
```

Comment:
Partials with methods can be problematic - see https://github.com/pcdshub/typhos/blob/55352823e41f7f6998052d9eaa54aa52bdc9c2dd/typhos/utils.py#L517

Since this probably lives throughout the life of the program, it's not likely an actual issue though

Reply:
I remembered this but didn't do anything about it because this kind of practice is already spread around the app in many places

Reply:
The most correct thing to do within normal qt bounds without juggling weak refs is to restructure this in terms of individual QObjects for each thing I want to do, right?

Reply:
Yeah, or having individual methods for what you want to do (such that it can be done without partials).

Reply:
In this case we dynamically create hundreds of rows that need this logic, so I can't make individual methods unfortunately- so it sounds like "make a class that does the thing" is the most succinct way to do it